src/components/CountryList.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import styles from './CountryList.module.css';
+
+import Country from './Country';
+
+const CountryList = ({ countries }) => (
+  <ul className="countryContainer">
```

Comment:
- [ ] To match the design guidelines on the details page, please make only one country in the line, the country name on the left, and the count on the right.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/176436863-3cd1e5cc-01d3-45e8-a8e4-cfb2a0b91c86.png) | ![image](https://user-images.githubusercontent.com/24703171/176436952-1e518de5-2273-4f18-bd54-faad871d71e6.png) |